python/adapters/hc_storage_adapter.py

```diff
@@ -68,32 +60,19 @@ def payments_output_schema():
             'source': str,
             'destination': str,
             'amount': int,
-            'memo': str,
+            'tx_order': int,
+            'tx_memo': str,
+            'tx_account': str,
+            'tx_account_sequence': int,
             'tx_fee': int,
             'tx_charged_fee': int,
-            'op_index': int,
             'tx_status': str,
-            'op_status': str,
             'tx_hash': str,
-            'timestamp': datetime
-        }
-
-    @staticmethod
-    def creations_output_schema():
-        """
-        :return: A dictionary of columns saved by the History collector. Key - name, Value - type
-        """
-
-        return {
-            'source': str,
-            'destination': str,
-            'starting_balance': int,
-            'memo': str,
-            'tx_fee': int,
-            'tx_charged_fee': int,
-            'op_index': int,
-            'tx_status': str,
+            'op_order': int,
```

Comment:
does this mean the order of the operation in the tx? if so, op index would be the correct term used in the blockchain

Reply:
Thanks, I didn't know there's a specific term in blockchain for that. I changed it based on a design document I got for this change, but I agree - it will make more sense to keep that as 'index', I'll change it